pyvista/core/filters/poly_data.py

```diff
@@ -2701,6 +2702,9 @@ def extrude_rotate(
                a value for this keyword argument to prevent future changes
                in behavior and warnings.
 
+        rotation_axis : numpy.ndarray or sequence
+            Direction vector of axis around which the rotation is done.
```

Comment:
We should put in an explicit note about compatibility with vtk version.

Reply:
```suggestion
            The direction vector of the axis around which the rotation is done.
            It requires vtk>=9.1.0.
```